journal/original/15-chapter-12.adoc

```diff
@@ -1592,6 +1635,9 @@ but they rejecting counsel, and slighting those visitations of love to them,
 I was moved now, before they were quite overturned, to lay their backsliding, hypocrisy,
 and treacherous dealing before them, thus:
 
+[.embedded-content-document.letter]
```

Comment:
I think, because the subject of this is the "several powers" (which I take to mean various political leaders), this would probably be better as a `address`